src/Integration.TeamExplorer.UnitTests/SectionControllerTests.cs

```diff
@@ -64,10 +61,14 @@ public void TestInitialize()
         #region Tests
 
         [TestMethod]
-        public void SectionController_Ctor_ArgChecks()
+        public void MefCtor_CheckIsExported()
```

Comment:
I didn't add the "check is singleton" or "check is non-shared" tests since we don't currently specify a `CreationPolicy` attribute on the class, and I wasn't sure what the correct value should be.